src/trans/common/internal/supolf_mod.F90

```diff
@@ -221,7 +222,7 @@ SUBROUTINE SUPOLF(KM,KNSMAX,DDMU,DDPOL,KCHEAP)
 
     DO JN=KM,KNSMAX,2
       DO JCORR=1,ICORR(JN)
-        DDPOL(JN)=DDPOL(JN)/ZSCALE
+        DDPOL(JN)=MAX(DDPOL(JN)/ZSCALE, ZEPS)
```

Comment:
I think this modification is equivalent to what is in the following lines.
Perhaps you can revert this modification.

Reply:
Wooooops I think this was an error!